node/src/accountant/scanners.rs

```diff
@@ -1975,10 +2468,10 @@ mod tests {
         transaction_receipt_1.transaction_hash = transaction_hash_1;
```

Comment:
I can't comment on the line directly, but that `let transaction_hash_1 = ` line above: is that a candidate for `make_tx_hash()`?